extensions/amp-link-rewriter/0.1/config-options.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {getChildJsonConfig} from '../../../src/json';
+import {hasOwn} from '../../../src/utils/object';
+import {
+  user,
+  userAssert,
+} from '../../../src/log';
+
+/**
+ * @param {!AmpElement} element
+ * @return {!{output: string, section:Array, attribute:Object, vars:Object}}
```

Comment:
Instead of an object. Could you please create a Typedef for the returned config object thanks.